### PR TITLE
Modify example scripts to run on Windows

### PR DESCRIPTION
**Pull request by cedrou**
The scripts `run-me.sh` of all examples for Marian have been slightly modified to be able to run on Windows Subsystem for Linux (WSL, aka "bash on Windows", see [this page](https://docs.microsoft.com/en-us/windows/wsl/install-win10)).

There are only 2 changes:
- Marian tools path are put into variables (`MARIAN_TRAIN`, `MARIAN_DECODER`, …) to allow adding the '.exe' suffix which is part of the binary name on Windows. This avoid cluttering in the rest of the script.
- As '.sh' files are not automatically bound to Bash on Windows, we need to explicitly call `bash ./scripts/validate.sh` in the flag `--valid-script-path` of marian.

### CODE REVIEW

**Comment by emjotde** [comment]
@cedrou The examples are meant to be run from within the Marian folder, for instance the most recent version of https://github.com/marian-nmt/marian has a gitmodule that pulls that in and places the `examples` folder in the `marian` folder. Can you make sure this works? The changed paths look like they might get in the way.

**Comment by cedrou** [comment]
The scripts should work like before. The only change I did was to move the ´/build’ part of the path that appeared at each tool call into the MARIAN variable.

File: transformer/run-me.sh [hunk 1]
```diff
@@ -79,13 +90,13 @@ then
 fi
 
 # find best model on dev set
-ITER=`cat model/valid.log | grep translation | sort -rg -k8,8 -t' ' | cut -f4 -d' ' | head -n1`
+ITER=`cat model/valid.log | grep translation | sort -rg -k12,12 -t' ' | cut -f8 -d' ' | head -n1`
```
Comment (snukky) [inline transformer/run-me.sh:93]: This looks like a proper bugfix.

File: transformer/run-me.sh [hunk 2]
```diff
@@ -79,13 +90,13 @@ then
...
 
 # translate test sets
 for prefix in test2014 test2015 test2016
 do
     cat data/$prefix.bpe.en \
-        | $MARIAN/build/marian-decoder -c model/model.npz.decoder.yml -m model/model.iter$ITER.npz -d $GPUS -b 12 -n \
+        | $MARIAN_DECODER -c model/model.npz.decoder.yml -m model/model.iter$ITER.npz -d $GPUS -b 12 -n -w 6000 \
```
Comment (snukky) [inline transformer/run-me.sh:99]: 10GB is used for training, so using 6GB here is also OK.

**Review by snukky — APPROVED** [review]
The changes are OK. On a UNIX machine `examples/training-basics` works from inside `marian-dev`, including validation steps.

File: transformer/run-me.sh (hunk 1 above)
Comment (cedrou) [reply transformer/run-me.sh:93]: Oh yes, this one is because the log format has changed

**Pull request merged**